ts/screens/main/MessagesScreen.tsx

```diff
@@ -23,15 +23,15 @@ import { MessageWithContentPO } from "../../types/MessageWithContentPO";
 type ReduxMappedProps = Readonly<{
   isLoadingMessages: boolean;
   messages: ReadonlyArray<MessageWithContentPO>;
-  services: ServicesState;
+  services: Readonly<ServicesState>;
```

Comment:
this is not needed since there's already `Readonly<>` at the `ReduxMappedProps` level